src/native/corehost/build.cmd

```diff
@@ -186,6 +186,9 @@ if "%__RuntimeFlavor%" NEQ "Mono" (
     echo Copying "%__CoreClrArtifacts%\corehost\PDB\singlefilehost.pdb"  "%__CMakeBinDir%/corehost/PDB/"
     copy /B /Y "%__CoreClrArtifacts%\corehost\PDB\singlefilehost.pdb"  "%__CMakeBinDir%/corehost/PDB/"
 
+    echo Embedding "%__CoreClrArtifacts%\mscordaccore.dll" into "%__CMakeBinDir%\corehost\singlefilehost.exe"
+    "%__CoreClrArtifacts%\dactabletools\InjectResource.exe" /bin:"%__CoreClrArtifacts%\mscordaccore.dll" /dll:"%__CMakeBinDir%\corehost\singlefilehost.exe" /name:MINIDUMP_AUXILIARY_EMBEDDED_MODULE
```

Comment:
The only thing I'd love is to be able to verify here that what we are embedding is actually 200-signed.

Reply:
That would be nice. Do we have tools examples for that?
We can also do it as a separate change if this PR could unblock further work.

Reply:
I assume something verifies mscordac.dll signature?

Reply:
Nothing really verifies it within the repo.

Reply:
I think we should merge this then and add verification when/if we figure how to verify.